code/game/machinery/computer/guntracker.dm

```diff
@@ -0,0 +1,96 @@
+/obj/machinery/computer/guntracker
```

Comment:
The control application should be a modular computer app (restricted to consoles) and not a standalone hardware.